toolkit/docs/building/prerequisites.md

```diff
@@ -1,28 +1,9 @@
 
 # Build Requirements
 
-## Requirements were validated on `Ubuntu 18.04`
+## CBL-Mariner
+Build Requirements on CBL-Mariner listed [here](./prerequisites-mariner.md)
```

Comment:
Nit-pick: please add periods at the end of sentences.

Reply:
Thanks for you advice.
Done!